system-parachains/asset-hubs/asset-hub-kusama/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -66,87 +66,91 @@ impl<T: frame_system::Config> WeightInfo<T> {
 		// Proof Size summary in bytes:
 		//  Measured:  `430`
 		//  Estimated: `6196`
-		// Minimum execution time: 145_729_000 picoseconds.
-		Weight::from_parts(152_880_000, 6196)
+		// Minimum execution time: 145_359_000 picoseconds.
+		Weight::from_parts(151_190_000, 6196)
 			.saturating_add(T::DbWeight::get().reads(7))
 			.saturating_add(T::DbWeight::get().writes(4))
 	}
+	/// Storage: `System::Account` (r:1 w:1)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
 	pub(crate) fn buy_execution() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
-		//  Estimated: `0`
-		// Minimum execution time: 720_000 picoseconds.
-		Weight::from_parts(900_000, 0)
+		//  Estimated: `3593`
+		// Minimum execution time: 8_010_000 picoseconds.
+		Weight::from_parts(8_609_000, 3593)
```

Comment:
@karolk91 @franciscoaguirre probably it is ok according to the configuration, but could you please double-check this big increase for `BuyExecution`? 
I see the https://github.com/paritytech/polkadot-sdk/pull/7944 has the same. So the setting `WeightLimit` causes this increase and additional `System::Account` R/W?

Reply:
> @karolk91 @franciscoaguirre probably it is ok according to the configuration, but could you please double-check this big increase for `BuyExecution`? I see the [paritytech/polkadot-sdk#7944](https://github.com/paritytech/polkadot-sdk/pull/7944) has the same. So the setting `WeightLimit` causes this increase and additional `System::Account` R/W?

the same for polkadot system chains: https://github.com/polkadot-fellows/runtimes/actions/runs/16959433981/job/48071163445#step:16:34